tests/smartswitch/platform_tests/test_show_platform_dpu.py

```diff
@@ -12,12 +12,12 @@
 from tests.common.devices.sonic import *  # noqa: 403
 
 pytestmark = [
-    pytest.mark.topology('t1')
+    pytest.mark.topology('smartswitch')
 ]
 
 
 def test_midplane_ip(duthosts, enum_rand_one_per_hwsku_hostname,
-                     platform_api_conn):
+                     platform_api_conn, check_smartswitch_and_dark_mode):
```

Comment:
Since the autouse of  check_smartswitch_and_dark_mode is True, we don't need to call it explicitly here.
I think all test can remove the argument of check_smartswitch_and_dark_mode

Reply:
Correct. I will remove it.